### PR TITLE
 this commit will add the getTopics feature and compled the 3rd ticket

### PR DESCRIPTION
**Pull request by ColemDev**

### CODE REVIEW

File: controllers/nc_news.controller.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const { selectTopics } = require("../models/nc-news.model.js");
+
+exports.getTopics = (req, res, next) => {
+  selectTopics()
+    .then((topics) => {
+      console.log(topics, "controller topics");
```
Comment (scar1377) [inline controllers/nc_news.controller.js:6]: remove console.logs

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.topics).toBeInstanceOf(Array);
+        response.body.topics.forEach((topic) => {
+          expect(topic.hasOwnProperty("slug")).toBe(true);
+          expect(topic.hasOwnProperty("description")).toBe(true);
+        });
+      });
+  });
```
Comment (scar1377) [inline __tests__/app.test.js:26]: Need a sad path - 404 path not found  e.g. /api/topiccccc

**Review by scar1377 — CHANGES_REQUESTED** [review]
Hi Michael. The code looks good, just need a sad path test

File: controllers/nc_news.controller.js (hunk 1 above)
Comment (ColemDev) [reply controllers/nc_news.controller.js]: Okay this is resolved now. Would I normally send a new pull request or would we just resolve this one?

File: __tests__/app.test.js (hunk 2 above)
Comment (ColemDev) [reply __tests__/app.test.js:26]: A sad path has been added and the code pushed.

**Review by scar1377 — APPROVED** [review]
All good to merge!

**Pull request merged**